exercises/en/test_05_22.py

```diff
@@ -0,0 +1,11 @@
+def test():
```

Comment:
please remove the disclaimer from this question as requested by mike 

this part -> "(Disclaimer: Much more than these two measurements are needed to determine if someone is healthy)"